src/utils/date.ts

```diff
@@ -83,9 +83,26 @@ export function secondsBetween(
   return differenceInSeconds(parsedEnd, parsedStart);
 }
 
-export function formatToClock(isoDate: string | Date, language: Language) {
-  const parsed = isoDate instanceof Date ? isoDate : parseISO(isoDate);
-  return formatLocaleTime(parsed, language);
+/**
+ * @param isoDate
+ * @param language
+ * @param roundingMethod rounding of minutes. Default: ignores seconds
+ * @returns
+ */
```

Comment:
I think it looks a little messy when not all parameters are documented. I think the parameter `roundingMethod` in itself is self-explanatory, but I see the value in describing the default behaviour. Maybe just make roundingMethod an obligatory parameter? I don't think it is too much to ask of callers of this function to be explicit of how they want the minute to be rounded.

Reply:
Made the param required, and since the the trip search result changes was merged, I added rounding there as well.